config.ts

```diff
@@ -25,5 +25,8 @@ module.exports = {
     atomicassets: {
         contract: 'atomicassets',
         collection: 'alien.worlds'
-    }
+    },
+    docs_host: null
 }
+
+export default config;
```

Comment:
converted config to an ESM module to allow es6 style  import, which is needed for Tap test to function well.